### PR TITLE
Fix Reddit Homepage

### PR DESCRIPTION
**Pull request by Kickball**
Update the Reddit entry to remove the duplicate home and demo link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -390,7 +390,7 @@ _[IPBX](https://en.wikipedia.org/wiki/IP_PBX) telephony software_
   * [phpBB](https://www.phpbb.com/) - Flat-forum bulletin board software solution that can be used to stay in touch with a group of people or can power your entire website. ([Source Code](https://github.com/phpbb/phpbb)) `GPLv2` `PHP`
   * [PPnet](https://github.com/pixelpark/ppnet) - Create and host your own social network. `MIT` `Javascript`
   * [Pump.io](http://pump.io/) - Stream server that does most of what people really want from a social network. ([Source Code](https://github.com/e14n/pump.io)) `Apache` `Nodejs`
-  * [Reddit](https://www.reddit.com/) - Social networking and news website where registered community members can submit content. ([Demo](https://www.reddit.com/), [Source Code](https://github.com/reddit/reddit)) `CPALv1` `Python`
+  * [Reddit](https://www.reddit.com/r/redditdev/) - Social networking and news website where registered community members can submit content. ([Demo](https://www.reddit.com/), [Source Code](https://github.com/reddit/reddit)) `CPALv1` `Python`
```
Comment (nodiscc) [inline README.md]: In the sidebar:

> A subreddit for discussion of reddit API clients.

I don't think this is appropriate (I see this is the homepage for the reddit organization on Github, but I still think it misses the point). You might as well revert the change, and remove the Demo link to conform to the no-duplicate rule

**Comment by evaryont** [comment]
Is reddit still appropriate for the list any more given [their announcement](https://www.reddit.com/r/changelog/comments/6xfyfg/an_update_on_the_state_of_the_redditreddit_and/) about archiving the open source code?

**Comment by nodiscc** [comment]
@evaryont As long as https://github.com/reddit/reddit stays under a Free Software license and is not [unmaintained since 6-12 months](https://github.com/Kickball/awesome-selfhosted/blob/master/.github/CONTRIBUTING.md), I see no reason to remove it. We'll see if a community fork emerges by then. Otherwise we can remove when if it reaches the 6/12 month inactivity threshold.

**Comment by nodiscc** [comment]
@Kickball, any updates?

**Comment by evaryont** [comment]
Before October 14th, Reddit hadn't committed anything to the repository since April of this year. Then at the beginning of the month they pushed a few commits that cleaned up the repository for it's archival status. Discounting those recent commits, there hasn't been any commits to the repository for 6 months.

Given the fact that the repository has been officially archived and the 6 month threshold has been hit (sorta), the link should probably be removed. A community fork can be re-added at a later time through the normal process.

**Comment by nodiscc** [comment]
> This repository is archived and will not receive any updates or accept issues or pull requests.

Indeed

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please rebase to get rid of the merge commits in this Pull Request</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>
